addOns/wappalyzer/src/main/java/org/zaproxy/zap/extension/wappalyzer/TechPanel.java

```diff
@@ -157,7 +160,7 @@ private javax.swing.JToolBar getPanelToolbar() {
             panelToolbar.setFloatable(false);
             panelToolbar.setRollover(true);
             panelToolbar.setPreferredSize(new java.awt.Dimension(800, 30));
-            panelToolbar.setFont(new java.awt.Font("Dialog", java.awt.Font.PLAIN, 12));
+            panelToolbar.setFont(new Font(Font.DIALOG, Font.PLAIN, 12));
```

Comment:
This should actually be using `FontUtils`, or drop the hardcoded sizes, which are most likely not needed as the defaults are good elsewhere. (same for other places)

Reply:
Good point, it probably doesn't need to be set at all. I'll check.